docs/docs/cmd/teams/membersettings/membersettings-list.md

```diff
@@ -55,3 +55,20 @@ m365 teams membersettings list --teamId 2609af39-7775-4f94-a3dc-0dd67657e900
     allowCreateUpdateChannels,allowCreatePrivateChannels,allowDeleteChannels,allowAddRemoveApps,allowCreateUpdateRemoveTabs,allowCreateUpdateRemoveConnectors
     1,1,1,1,1,1
     ```
+
+=== "Markdown"
+
+    ```md
+    # teams membersettings list --teamId "2609af39-7775-4f94-a3dc-0dd67657e900"
+
+      Date: 5/7/2023
+
+      Property | Value
+      ---------|-------
+      allowCreateUpdateChannels | true
+      allowCreatePrivateChannels | true
+      allowDeleteChannels | true
+      allowAddRemoveApps | true
+      allowCreateUpdateRemoveTabs | true
+      allowCreateUpdateRemoveConnectors | true
```

Comment:
similar problem in the render on the page 
![image](https://github.com/pnp/cli-microsoft365/assets/58668583/ca59e80e-0f5c-40ef-b6ae-7a14d0afad0a)